table/src/test/java/com/bazaarvoice/emodb/table/db/astyanax/StashBlackListTableTest.java

```diff
@@ -0,0 +1,56 @@
+package com.bazaarvoice.emodb.table.db.astyanax;
+
+import com.bazaarvoice.emodb.sor.api.Intrinsic;
+import com.bazaarvoice.emodb.sor.condition.Conditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class StashBlackListTableTest {
+
+    @Test
+    public void testisTableBlacklisted() throws Exception {
+
+        // true cases
+        Assert.assertEquals(AstyanaxTableDAO.isTableBlacklisted("review:customer", "sor-ugc", Conditions.alwaysTrue()), true);
+
+        Assert.assertEquals(AstyanaxTableDAO.isTableBlacklisted("review:customer", "sor-ugc", Conditions.intrinsic(Intrinsic.TABLE, Conditions.equal("review:customer"))), true);
+        Assert.assertEquals(AstyanaxTableDAO.isTableBlacklisted("review:customer", "sor-ugc", Conditions.intrinsic(Intrinsic.TABLE, Conditions.like("*:customer"))), true);
+        Assert.assertEquals(AstyanaxTableDAO.isTableBlacklisted("review:customer", "sor-ugc",
+                Conditions.or(Conditions.intrinsic(Intrinsic.TABLE, Conditions.equal("review:customer")), Conditions.intrinsic(Intrinsic.TABLE, Conditions.equal("review:customer2")))), true);
+
+        Assert.assertEquals(AstyanaxTableDAO.isTableBlacklisted("review:customer", "sor-ugc", Conditions.intrinsic(Intrinsic.PLACEMENT, Conditions.equal("sor-ugc"))), true);
+        Assert.assertEquals(AstyanaxTableDAO.isTableBlacklisted("review:customer", "sor-ugc", Conditions.intrinsic(Intrinsic.PLACEMENT, Conditions.like("*sor*"))), true);
+        Assert.assertEquals(AstyanaxTableDAO.isTableBlacklisted("review:customer", "sor-ugc",
+                Conditions.or(Conditions.intrinsic(Intrinsic.PLACEMENT, Conditions.equal("sor-ugc")), Conditions.intrinsic(Intrinsic.PLACEMENT, Conditions.equal("sor-cat")))), true);
+
+        Assert.assertEquals(AstyanaxTableDAO.isTableBlacklisted("review:customer", "sor-ugc",
+                Conditions.and(Conditions.intrinsic(Intrinsic.TABLE, Conditions.equal("review:customer")), Conditions.intrinsic(Intrinsic.PLACEMENT, Conditions.equal("sor-ugc")))), true);
+        Assert.assertEquals(AstyanaxTableDAO.isTableBlacklisted("review:customer", "sor-ugc",
+                Conditions.and(Conditions.or(Conditions.intrinsic(Intrinsic.TABLE, Conditions.equal("review:customer")), Conditions.intrinsic(Intrinsic.TABLE, Conditions.equal("review:customer2"))),
+                        Conditions.intrinsic(Intrinsic.PLACEMENT, Conditions.equal("sor-ugc")))), true);
```

Comment:
As noted previously, these unit tests miss some conditions, particularly around tables metadata.  For example, assume we want to exclude all tables with the table metadata "type" equal to "review".  The above tests would fail for the condition:
```
{..,"type":"review"}
```